File: docker-files/jwt.js
const jwt = require('jsonwebtoken');

const token = jwt.sign(
  {
    mercure: {
      publish: ['*'], // or subscribe: ['*']
    },
  },
  'nQp2WGR2YxJRHqbwl6N0FX+0NOgcnTR5vavVcQ9+VNk=',
  { algorithm: 'HS256' }
);

console.log(token);

File: docker-files/jwt.py
import jwt

payload = {
    "mercure": {
        "publish": ["*"]
    }
}

secret = "nQp2WGR2YxJRHqbwl6N0FX+0NOgcnTR5vavVcQ9+VNk="
token = jwt.encode(payload, secret, algorithm="HS256")
print(token)

